src/components/Personalization/handlers/createProcessPropositions.js

```diff
@@ -15,17 +15,32 @@ export default ({ schemaProcessors, logger }) => {
     return Promise.resolve()
       .then(render)
       .then(() => {
-        if (logger.enabled) {
-          logger.info(`Action ${item.toString()} executed.`);
-        }
+        logger.logOnContentRendering({
+          status: "rendering-succeeded",
+          detail: {
+            propositionDetails: item.getProposition().getNotification(),
+            item: item.toJSON()
+          },
+          message: `Action ${item.toString()} successfully executed.`,
+          logLevel: "info",
+        });
+
```

Comment:
It's a bit confusing that the rendering-started message is for all page or view propositions, but then the rendering-succeeded and rendering-failed messages are for each item. Is this a compromise? Or is this how Target did things?

Reply:
Target it's based on a mbox. In Web SDK we don't have the concept of an mbox.